google-cloud-examples/src/main/java/com/google/cloud/examples/storage/snippets/StorageSnippets.java

```diff
@@ -1126,6 +1126,39 @@ public Bucket setDefaultKmsKey(String bucketName, String kmsKeyName) throws Stor
     return bucket;
   }
 
+  /** Example of displaying Bucket metadata */
+  public void getBucketMetadata(String bucketName) throws StorageException {
+    // [START storage_get_bucket_metadata]
+    Storage storage = StorageOptions.getDefaultInstance().getService();
+
+    // The name of a bucket, e.g. "my-bucket"
+    // String bucketName = "my-bucket";
+
+    // Select all fields
+    // Fields can be selected individually e.g. Storage.BucketField.NAME
+    Bucket bucket = storage.get(bucketName, BucketGetOption.fields(Storage.BucketField.values()));
+
+    // Print bucket metadata
+    System.out.println("BucketName: " + bucket.getName());
+    System.out.println("DefaultEventBasedHold: " + bucket.getDefaultEventBasedHold());
+    System.out.println("DefaultKmsKeyName: " + bucket.getDefaultKmsKeyName());
+    System.out.println("Id: " + bucket.getGeneratedId());
+    System.out.println("IndexPage: " + bucket.getIndexPage());
+    System.out.println("Labels: " + bucket.getLabels());
```

Comment:
This is a Map. How will this be handled?

Reply:
It's a Map<String, String>, so the toString function works just fine here, would look like "{k1=v1, k2=v2}"

Reply:
Gotcha, thanks for clarifying. Please handle the map through an iterator so the data type is surfaced.